DittoHealth/build.gradle

```diff
@@ -22,6 +22,7 @@ dependencies {
     implementation libs.androidx.compose.ui.uiTooling
     implementation libs.androidx.compose.ui.uiToolingPreview
     implementation libs.androidx.navigation.navigationCompose
+    implementation project(path: ':DittoHealthMetrics')
```

Comment:
was this just for testing?

Reply:
That was to include the Health metrics into the .aar for the tool.